tests/posix/semaphores/src/main.c

```diff
@@ -10,6 +10,7 @@
 #include <pthread.h>
```

Comment:
I have no objections to the change but could you elaborate in commit msg why this change is needed, now it just states the obvious thing (what is being changed).

Reply:
Sure, I can update.